packages/pouchdb-adapter-asyncstorage/src/bulk_docs.js

```diff
@@ -53,35 +54,59 @@ export default function (db, req, opts, callback) {
         })
       }
 
-      let binData
+      let resolveBinaryData;
```

Comment:
Having a lot of trouble getting NPM to install all the dependencies for this project.  It keeps getting stuck in a circular process of loading things, any ideas?